bin/user_config_converter.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+from enum import Enum, auto
+import importlib
+import json
+import os
+from pathlib import Path
+import sys
+from typing import Type
+
+from fusion_engine_client.messages.configuration import PlatformStorageDataMessage
+from fusion_engine_client.parsers.decoder import FusionEngineDecoder
+
+# Add the parent directory to the search path to enable p1_runner package imports when not installed in Python.
+repo_root = Path(os.path.dirname(__file__)).parent.absolute()
+sys.path.append(str(repo_root))
+
+# Example version imported just to help with type checking
```

Comment:
"checking. The UserConfig class will be overridden below with the appropriate platform type and config version based on the inptu file."